src/light.h

```diff
@@ -54,11 +54,12 @@ inline u8 diminish_light(u8 light, u8 distance)
 
 inline u8 undiminish_light(u8 light)
 {
+	assert(light <= LIGHT_SUN);
 	// We don't know if light should undiminish from this particular 0.
 	// Thus, keep it at 0.
 	if (light == 0)
 		return 0;
-	if (light == LIGHT_MAX)
+	if (light == LIGHT_SUN)
```

Comment:
Seems a change of behaviour, one is 14 the other 15.
How about `if (light >= LIGHT_MAX)`?

Reply:
Updated.